Packs/FeedAutofocus/Integrations/FeedAutofocus/FeedAutofocus.py

```diff
@@ -86,13 +120,138 @@ def http_request(self, feed_type) -> list:
                 method="GET",
                 url=url,
                 verify=self.verify,
-                headers=self.headers
+                headers=headers
             )
             res.raise_for_status()
             indicator_list.extend(res.text.split('\n'))
 
         return indicator_list
 
+    def sample_http_request(self) -> list:
+        """The HTTP request for the sample feed.
+
+        Args:
+
+        Returns:
+            list. A list of indicators fetched from the feed.
+        """
+        request_body = {
+            'apiKey': self.api_key,
+            'artifactSource': 'af',
+            'scope': self.scope_type,
+            'query': self.sample_query,
+            'type': 'scan',
+        }
+
+        initiate_sample_res = requests.request(
+            method="POST",
+            headers={'Content-Type': "application/json"},
+            url=SAMPLE_FEED_REQUEST_BASE_URL,
+            verify=self.verify,
+            json=request_body
+        )
+        initiate_sample_res.raise_for_status()
+
+        af_cookie = initiate_sample_res.json()['af_cookie']
+        time.sleep(20)
```

Comment:
why?

Reply:
The api needs some time to process the request, thats a random number that seemed to be working at all times, lets talk about it tomorrow morning